lms/templates/edxnotes/edxnotes.html

```diff
@@ -26,7 +26,7 @@ <h1 class="page-title">
         <div class="wrapper-notes-search">
           <form role="search" action="${search_endpoint}" method="GET" id="search-notes-form" class="is-hidden">
             <label for="search-notes-input" class="sr">${_('Search notes for:')}</label>
-            <input type="search" class="search-notes-input" id="search-notes-input" name="note" placeholder="${_('Search notes for...')}">
+            <input type="search" class="search-notes-input" id="search-notes-input" name="note" placeholder="${_('Search notes for...')}" required aria-required="true">
```

Comment:
@clrux, @cptvitamin said to add both as a failsafe since not all assistive tech leverages ``required`` yet.